process/block/preprocess/basePreProcess.go

```diff
@@ -329,6 +329,8 @@ func (bpp *basePreProcess) computeGasConsumed(
 	if bpp.shardCoordinator.SelfId() == senderShardId {
 		gasConsumedByTxInSelfShard = gasConsumedByTxInSenderShard
 
+		// the max gas limit to be compared with, should be the maximum value between max gas limit per mini block and max gas limit per tx. If max gas limit per tx
```

Comment:
does this comment apply here?

Reply:
Partially yes, I have adjusted the comment a little bit